plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -101,6 +101,18 @@ class BasicsChart extends React.Component {
     this.setSectionsToState();
   }
 
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
eslint is breaking the test on this
```suggestion
  // eslint-disable-next-line camelcase
  UNSAFE_componentWillReceiveProps(nextProps) {
```

Reply:
Thanks